pkg/linode-bs/deploy/releases/linode-blockstorage-csi-driver.yaml

```diff
@@ -269,7 +269,11 @@ data:
     id="$(kubectl get node/"${NODE_NAME}" -o jsonpath='{.spec.providerID}')"
     if [[ ! -z "${id}" ]]; then
       echo "${id}"
-      echo -n "${id:9}" > /linode-info/linode-id
+      if [[  "${id}" =~ "linode://" ]]; then
```

Comment:
Could do `if [[ "${id}" == "linode://"* ]]; then` to specifically check for prefix, but I'm fine with this too